website/docs/docs/build/metricflow-time-spine.md

```diff
@@ -230,3 +259,52 @@ where date_day > dateadd(year, -4, current_timestamp())
 and date_hour < dateadd(day, 30, current_timestamp())
 ```
 </File>
+
+
+## Custom calendar
+
+<VersionBlock lastVersion="1.8">
+
+Being able to configure custom calendars, such as like a fiscal calendar, is available in [dbt Cloud Versionless](/docs/dbt-versions/upgrade-dbt-version-in-cloud#versionless) or dbt Core [v1.9 and above](/docs/dbt-versions/core). 
+
+To access this feature, [upgrade to Versionless](/docs/dbt-versions/versionless-cloud) or dbt Core v1.9 and above.
+</VersionBlock>
+
+<VersionBlock firstVersion="1.9">
+
+Custom date transformations can be complex, and organizations often have unique needs that can’t be easily generalized. Creating a custom calendar model allows you to define these transformations in SQL, offering more flexibility than native transformations in MetricFlow. This approach lets you map custom columns back to MetricFlow granularities, ensuring consistency while giving you control over the transformations.
+
+For example, if you use a custom calendar in your organization, such as a fiscal calendar, you can configure it in MetricFlow using its date and time operations. 
+
+- This is useful for calculating metrics based on a custom calendar, such as fiscal quarters or weeks. 
+- Use the `custom_granularities` key to define a non-standard time period for querying data, such as a `retail_month` or `fiscal_week`, instead of standard options like `day`, `month`, or `year`. 
+- This feature provides more control over how time-based metrics are calculated.
+
+### Calendar model requirements
+
+To use a custom calendar model, ensure it meets the following requirements:
+
+- The model must be at a daily grain and joinable to other date columns using `date_day`.
+- Columns must align with expected data types for a given granularity (for example, `quarter` should be a date).
```

Comment:
Can we update this to be a little more specific:
"The `standard_granularity_column` must be a date time type."
(Or something similar - datetime/date/timestamp I think are all fine but not sure if I'm missing options from any specific engines)
Also could move that note to the section about `standard_granularity_column`.
The other columns can be any type so I don't think we need to add any detail about that.